Sources/MapboxNavigation/Constants.swift

```diff
@@ -110,3 +110,8 @@ public typealias CarPlayUserInfo = [String: Any?]
  to a color defined in `InstructionsCardContainerView.highlightedBackgroundColor`.
  */
 let InstructionCardHighlightDistance: CLLocationDistance = 152.4 // 500 ft
+
+/**
+ Default tolerance value used for configuring the underlying map source of route line, maneuver arrow and restricted areas.
+ */
+let RouteSourcesTolerance: Double = 0.375
```

Comment:
Not sure that breaking the the default out into a separate constant is adding much clarity in this case. Do you envision us needing to reuse this constant anywhere, in which case maybe it would need to be made public?

Reply:
I don't envision its reuse, so made it inline.